apis/console/src/macros.rs

```diff
@@ -0,0 +1,50 @@
+/*! Runtime debugging tools. */
+
+#[macro_export]
+/// Prints the expression to the console, does not add a newline
+macro_rules! print {
+    ($($arg:tt)*) => ({
+        use libtock::console::Console;
+        use core::fmt::Write;
```

Comment:
We shouldn't be importing types/traits into the caller's scope.

Reply:
We aren't, the bracket in `({` keeps them inside a dedicated scope. I actually hit that while testing, so it's tested.

Reply:
Oh I missed the bracket, thanks.